api/StorageManager.json

```diff
@@ -5,34 +5,56 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/StorageManager",
         "support": {
           "webview_android": {
```

Comment:
Since I've suggested other changes, may as well sort the browsers too

Reply:
Good idea, I just did that in 4125b1a.